source/plugin/modularization.rst

```diff
@@ -0,0 +1,61 @@
+===================
+Code Modularization
+===================
+
+When writing Sponge plugins, you may find that you need to modularize your code. Maybe you want to split your core into
+another project so that others can make use of it, or maybe you want to utilize a service another plugin provides. With
+Sponge, begone the days of shading or shadowing entire plugins into your plugin.
```

Comment:
For someone who doesn’t use Gradle, Shading is more obvious. For someone who doesn’t use Maven, Shadowing is more obvious.
> On Jul 15, 2016, at 9:16 PM, Robert Herhold <notifications@github.com> wrote:
> 
> In source/plugin/modularization.rst <https://github.com/SpongePowered/SpongeDocs/pull/516#discussion_r71059312>:
> 
> > @@ -0,0 +1,61 @@
> > +===================
> > +Code Modularization
> > +===================
> > +
> > +When writing Sponge plugins, you may find that you need to modularize your code. Maybe you want to split your core into
> > +another project so that others can make use of it, or maybe you want to utilize a service another plugin provides. With
> > +Sponge, begone the days of shading or shadowing entire plugins into your plugin.
> "gone are" would be more appropriate than "begone". Also, there's no need for "or shadowing" since shading and shadowing mean the same thing.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/SpongePowered/SpongeDocs/pull/516/files/5c3f2857eee9da4d9872a9dd142bf7d1971511e1#r71059312>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AF2RQnGCEvW36yU-ThZmgAL_fimuc78nks5qWD7xgaJpZM4JN2iL>.
>